src/main/java/edu/project1/views/GameView.java

```diff
@@ -0,0 +1,85 @@
+package edu.project1.views;
+
+import edu.project1.GameState;
+import edu.project1.Settings;
+import edu.project1.models.GameModel;
+import edu.project1.views.base.Coordinate;
+import edu.project1.views.base.Pixel;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * The {@code GameView} class is responsible for rendering the visual elements and messages of the game to the screen.
+ * It includes views for the hangman, secret word, game description, and alphabet for letter selection.
+ */
+public class GameView extends View {
+    private final HangmanView hangmanView;
+    private final SecretWordView secretWordView;
+    private final DescriptionView descriptionView;
+    private final AlphabetView alphabetView;
+    private final static Logger LOGGER = LogManager.getLogger();
+
+    /**
+     * Constructs a {@code GameView} with predefined coordinates for the game screen.
+     */
+    public GameView() {
+        super(
+            new Coordinate(0, 0),
+            new Coordinate(GLOBAL_WIDTH - 1, GLOBAL_HEIGHT - 1)
+        );
+
+        this.screen = new Pixel[GLOBAL_WIDTH][GLOBAL_HEIGHT];
+
+        this.hangmanView = new HangmanView(this.screen);
+        this.secretWordView = new SecretWordView(this.screen);
+        this.descriptionView = new DescriptionView(this.screen);
+        this.alphabetView = new AlphabetView(this.screen);
+    }
+
+    /**
+     * Renders the game based on the provided {@link GameModel}, updating the visual elements and displaying messages.
+     *
+     * @param model The {@link GameModel} containing the game's data and state.
+     */
+    @Override
+    public void render(GameModel model) {
+        if (!(model.getState() == GameState.FINISH)) {
+            LOGGER.info(Settings.SESSION_SEPARATOR);
+
+            hangmanView.render(model);
+            secretWordView.render(model);
+            descriptionView.render(model);
+            alphabetView.render(model);
```

Comment:
Вместо последовательных вызовов одного и того же метода render можно воспользоваться полиморфизмом. Вместо инициализации отдельных полей в конструкторе можно создать `List<View> views`, из которого потом вызывать метод render.

 ```suggestion
             views.forEach(view -> view.render(model));
 ```

Reply:
Да, действительно.
Исправил.